diff --git a/node_modules/react-native-text-recognition/android/build.gradle b/node_modules/react-native-text-recognition/android/build.gradle
index 73a3910..7986988 100644
--- a/node_modules/react-native-text-recognition/android/build.gradle
+++ b/node_modules/react-native-text-recognition/android/build.gradle
@@ -21,11 +21,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('TextRecognition_compileSdkVersion', 29)
-    buildToolsVersion safeExtGet('TextRecognition_buildToolsVersion', '29.0.2')
+    compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : 35
+    buildToolsVersion '35.0.0'
     defaultConfig {
-        minSdkVersion safeExtGet('TextRecognition_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('TextRecognition_targetSdkVersion', 29)
+        minSdkVersion 21
+        targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : 35
         versionCode 1
         versionName "1.0"
 
